src/test/groovy/wolox/bootstrap/services/InformationLoggingServiceSpec.groovy

```diff
@@ -0,0 +1,37 @@
+package wolox.bootstrap.services
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.context.annotation.ComponentScan
+import spock.lang.Specification
+import wolox.bootstrap.repositories.LogRepository
+
+@DataJpaTest
+@ComponentScan
+class InformationLoggingServiceSpec extends Specification {
+    @MockBean
+    LogRepository logRepository;
+
+    @Autowired
+    InformationLoggingService service;
```

Comment:
I think you can do 

```
def logRepository = Mock(LogRepository)
def service = new InformationLoggingService
```

Reply:
I'm not so sure about those modifications. LogRepository should be resolved by Spring Boot by autowiring, because it would be only one type of LogRepository. Anyway, I changed this to use @SpringBean annotation to mock that repository, as recommended by Spock documentation:

http://spockframework.org/spock/docs/1.3/modules.html#_using_code_springbean_code